src/Kerbalism/UI/Monitor.cs

```diff
@@ -610,10 +610,8 @@ void Indicator_signal(Panel p, Vessel v, Vessel_info vi)
 			ConnectionInfo conn = vi.connection;
 			bool remotetech = RemoteTech.Enabled;
 
-			// signal strength or when using RemoteTech signal delay
-			string signal_str = remotetech ?
-				!conn.linked ? "----" : conn.strength > Double.Epsilon ? KSPUtil.dateTimeFormatter.PrintTimeStampCompact(conn.strength) : Localizer.Format("#KERBALISM_Generic_NONE") :
-				Lib.HumanReadablePerc(conn.strength, "F2");
+			// signal strength
+			string signal_str = Lib.HumanReadablePerc(conn.strength, "F2");
```

Comment:
I think
 `string signal_str = conn.strength > Double.Epsilon ? Lib.HumanReadablePerc(conn.strength, "F2") : Localizer.Format("#KERBALISM_Generic_NONE");`
would be better

Reply:
Or maybe `string signal_str = conn.strength > Double.Epsilon ? Lib.HumanReadablePerc(conn.strength, "F2") : Lib.Color("#ffaa00", Lib.Italic(Localizer.Format("#KERBALISM_Generic_NO")));`